Makefile.prow

```diff
@@ -1,92 +1,87 @@
-# warning: verify this inside a container with following:
-# docker run -it --rm -v `pwd`:`pwd` -w `pwd` golang:1.18 bash -c "make -f Makefile.prow ci"
-# 
-# otherwise unintended system changes may include, replacing your currently installed kubectl, binaries in GOBIN, files in GOPATH/src
-# 
-# If formatting needs updating you can run
+# Warning: Verify this inside a container with the following command:
+# podman run -it --rm -v `pwd`:`pwd`:Z -w `pwd` golang:1.22 bash -c "make -f Makefile.prow ci"
```

Comment:
I think makes more sense to use same image used by CI https://github.com/openshift/release/blob/master/ci-operator/config/openshift/velero/openshift-velero-konveyor-dev.yaml#L5

Reply:
You are right I will add the following two (I used as well :Z to ensure code is readable with SELinux):
```
# podman run -it --rm -v `pwd`:`pwd`:Z -w `pwd` quay.io/konveyor/builder:ubi9-v1.22.2 bash -c "make -f Makefile.prow ci"
# podman run -it --rm -v `pwd`:`pwd`:Z -w `pwd` quay.io/konveyor/builder:ubi9-v1.22.2 bash -c "make -f Makefile.prow lint"
```